e2e/cypress/integration/markdown/markdown_image_spec.js

```diff
@@ -46,13 +47,33 @@ describe('Markdown', () => {
         // * Verify that HTML Content is correct.
         // Note we use the Gigantic timeout to ensure that the large images will load
         cy.getLastPostId().then((postId) => {
-            cy.get(`#postMessageText_${postId}`).find('p').
-                should('have.text', 'GitHub favicon:  ');
+            cy.get(`#postMessageText_${postId}`).find('div').
+                should('have.text', 'GitHub favicon:  ').
+                and('have.class', 'style--none');
 
             cy.get(`#postMessageText_${postId}`).find('img').
                 should('have.class', 'markdown-inline-img').
+                and('have.class', 'markdown-inline-img--hover').
+                and('have.class', 'cursor--pointer').
+                and('have.class', 'a11y--active').
                 and('have.attr', 'alt', 'Github').
                 and('have.attr', 'src', `${baseUrl}/api/v4/image?url=https%3A%2F%2Fgithub.githubassets.com%2Ffavicon.ico`);
         });
     });
+
+    it('opens preview window when image is clicked', () => {
+        // #  Post markdown message
+        cy.postMessageFromFile('markdown/markdown_inline_images_2.md');
+
+        // Note we use the Gigantic timeout to ensure that the large images will load
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`, {timeout: TIMEOUTS.GIGANTIC}).then((post) => {
+                // # Get the image and simulate a click.
+                cy.wrap(post).find('img.markdown-inline-img').first().click();
+
+                // * Verify that the file preview opens
+                cy.get('div.file-details__container').first().should('exist');
+            });
+        });
+    });
 });
```

Comment:
Please add test case based from original issue which is posting `![test image](https://www.mattermost.org/wp-content/uploads/2016/03/logoHorizontal.png)` and verifying expected behavior.

Reply:
Sure, I can use that one